deluge/core/core.py

```diff
@@ -1017,6 +1017,35 @@ def create_torrent(
             ),
         ).start()
 
+    @export
+    def create_torrent_sync(
```

Comment:
If I'm not mistaken we don't need a new method, create_torrent can return a deferred without affecting existing client usage

Reply:
I updated the PR to return a Deferred directly from the existing function. Looks like you were right.